python/xorbits/core/adapter.py

```diff
@@ -159,6 +164,90 @@ def wrapped(self: DataRef, *args, **kwargs):
     return wrapped
 
 
+def wrap_cls_method(
+    cls: Type,
+    func_name: str,
+    get_output_type,
+    pandas_or_numpy,
```

Comment:
Is there a way to pass a function or sth instead of a boolean `pandas_or_numpy`? Assume we have sklearn, this argument may not work.